tools/wptrunner/wptrunner/browsers/servodriver.py

```diff
@@ -5,7 +5,7 @@
 
 from mozprocess import ProcessHandler
 
-from serve.serve import make_hosts_file
+from tools.serve.serve import make_hosts_file
```

Comment:
In principle wptrunner should work when installed, so we probably can't rely on `tools` being available. It should definitely have `serve` in the import path, though… :/